libs/sistema/theme/tokens/_palette.tokens.scss

```diff
@@ -5,11 +5,12 @@ $color-dark-stronger: hsl(0, 0%, 5%);
 
 $color-light-stronger: #fff;
 
-$color-neutral-strong:    hsl(0, 0%, 44%);
-$color-neutral-regular:   hsl(0, 0%, 77%);
-$color-neutral-light:     hsl(0, 0%, 90%);
-$color-neutral-lighter:   hsl(0, 0%, 94%);
-$color-neutral-lightest:  hsl(240, 7%, 97%);
+$color-neutral-regular:   hsl(0, 0%, 44%);
```

Comment:
I have two questions:
- at some point, should this variable renaming be done also in non angular apps like demo pages and widgets ?
- I see that $color-neutral-light instances are not replaced by $color-neutral-lighter, Is it correct ?